### ISSUE TITLE
Frontend Team Meeting : Final Meeting Before the Customer Presentation 

### ISSUE THREAD

**Issue by yagmurse** [bounswe/2021SpringGroup4#245]
A meeting will be held on 3 January 2022 (03/01/2022) at 22:00.
Participants : Frontend team ( Yiğit Sarıoğlu, Yağmur Selek, Ali Alperen Durak)
Backend Team (Tolga Kerimoğlu)
Place: Google Meet

Our task: All functionalities that has been implemented will be revised and a demo presentation will be made

Topics to be revised:

Comment Functionality
Equipment Post & Equipment Cards & Equipment Search
Event Page Completion & Participants
Badge Implementation
List the badges

**Comment by yagmurse** [comment]
We held the meeting about 5 hours long. During the meeting we evaluated the completed and in progress tasks written in the description (Topics to be discussed). Everyone in frontend team and Tolga from the android team joined the meeting. We did peer writing for some bugs and manually merged the codes. We went over our scenario and did some tests.  Everything has been done can be found [under this commit](https://github.com/bounswe/2021SpringGroup4/commit/38277eccdc80d05ba747262ce63f28a31b0d7a64)
